test/t8n/t8n.cpp

```diff
@@ -128,6 +132,12 @@ int main(int argc, const char* argv[])
                 }
             }
 
+            if (block_reward.has_value())
+                state.touch(block.coinbase).balance += *block_reward;
```

Comment:
This should be done _before_ any transaction execution.

Reply:
I removed it. It looks that now previously failing test passed. I will check why and let know.

Reply:
Are you sure? Moving block reward to coinbase before transaction made a lot of test started failing.

Reply:
Should be _after_